source/comInterfaces_sconscript

```diff
@@ -26,18 +30,62 @@ def new_my_import(fullname):
 	return old_my_import(fullname)
 comtypes.client._generate._my_import = new_my_import
 
-def interfaceAction(target,source,env):
-	clsid=env.get('clsid')
+def makeIDEFriendly(path:str) -> None:
+	"""
+	Add a local import of * so that tools and IDE's can find definitions.
+	Prefer to import from comtypes.gen, at runtime behavior will not have changed.
+	Make code like:
+	``` python
+	from comtypes.gen import _C523F390_9C83_11D3_9094_00104BD0D535_0_3_0
+	globals().update(_C523F390_9C83_11D3_9094_00104BD0D535_0_3_0.__dict__)
+	__name__ = 'comtypes.gen.AcrobatAccessLib'
+	```
+	Become:
+	``` python
+	try:
+		from comtypes.gen import _C523F390_9C83_11D3_9094_00104BD0D535_0_3_0
+	except ModuleNotFoundError:
+		from _C523F390_9C83_11D3_9094_00104BD0D535_0_3_0 import *
+	globals().update(_C523F390_9C83_11D3_9094_00104BD0D535_0_3_0.__dict__)

```

Comment:
This example is missing an import in the except block, and therefore does not reflect what it really does below.

Reply:
Yep. I think I can remove this example, I don't think it really helps to clarify anything now.